frontend/widgets/SwellWelcome/components/LoggedOutView/style.js

```diff
@@ -0,0 +1,72 @@
+import { css } from 'glamor';
+
+/**
+ * @param {string} color font color for widget
+ * @param {string} background background for widget
+ * @param {string} margin margin value for widget
+ * @returns {Function}
+ */
+const container = (color, background, margin) => css({
+  color,
+  background,
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  height: '450px',
+  width: '100%',
+  textAlign: 'center',
+  margin,
+}).toString();
+
+const title = css({
+  maxWidth: '315px',
+  fontSize: '70px',
+  fontWeight: 'bold',
+  lineHeight: '1',
+}).toString();
+
+const subtext = css({
+  maxWidth: '315px',
+  fontSize: '16px',
+  marginBottom: '30px',
+}).toString();
+
+const joinButton = css({
+  fontSize: '18px',
+  marginBottom: '10px',
+  backgroundColor: '#002554',
```

Comment:
this should be a config too. didn't see this in prev review, sorry.

Reply:
made configurable